src/main/java/com/yellow/ordermanageryellow/Service/ProductCategoryService.java

```diff
@@ -1,21 +1,44 @@
 package com.yellow.ordermanageryellow.Service;
+import Exceptions.ObjectAlreadyExistException;
 import com.yellow.ordermanageryellow.Dao.ProductCategoryRepository;
 import com.yellow.ordermanageryellow.model.ProductCategory;
-import com.yellow.ordermanageryellow.model.Users;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.CommandLineRunner;
+import org.springframework.dao.EmptyResultDataAccessException;
 import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
 
 @Service
-public class ProductCategoryService implements CommandLineRunner {
+public class ProductCategoryService  {
     private final ProductCategoryRepository ProductCategoryRepository;
```

Comment:
the variable should start with a lower latter 
    private final ProductCategoryRepository productCategoryRepository;